tests/infra/docker_remote.py

```diff
@@ -190,10 +197,10 @@ def start(self):
         ]["IPAddress"]
         for _, rpc_interface in self.host.rpc_interfaces.items():
             rpc_interface.public_host = self.container_ip
-        self.remote.hostname = self.container_ip
+        self.hostname = self.container_ip
         LOG.debug(f"Started container {self.container_name} [{self.container_ip}]")
 
-    def get_host(self):
+    def get_rpc_host(self):
         return self.container_ip
```

Comment:
Is this used anywhere? Same for the one in `CCFRemote`.

Even grepping the current code base it seems this was dead code already.